cacheServer/UpdateTables.js

```diff
@@ -0,0 +1,48 @@
+
+//
+// Add ministries field to campus collection
+//
+iterator = db.ministries.find({}, {"_id": 1, "campuses": 1});
+while ( iterator.hasNext() ) {
+    ministry = iterator.next();
+
+    ministryId = ministry["_id"];
+    ministryCampuses = ministry["campuses"];
+
+    if (ministryCampuses.length == 0) {
+        print("Ministry " + ministryId +  " has no campuses.");
```

Comment:
in future make an error message instead of print outs 
---counts as a test to see if DB is correct